jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-soak.yaml

```diff
@@ -0,0 +1,170 @@
+- job-template:
+    builders:
+    - shell: |
+        # TODO(fejta): consider a stable tag instead of master
+        git clone https://github.com/kubernetes/test-infra -b master
+        './test-infra/jenkins/bootstrap.py' --job='{job-name}' --bare --root="${{GOPATH}}/src/k8s.io"
+    disabled: false
+    name: 'ci-{soak-suffix}'
+    node: 'e2e'
+    properties:
+    - build-blocker:
+        blocking-jobs:
+        - '{blocker}'
+        queue-scanning: '{scan}'
+        use-build-blocker: true
+    - build-discarder:
+        days-to-keep: 7
+    triggers:
+    - timed: '{frequency}'
+    wrappers:
+    - e2e-credentials-binding
+    - inject:
+        properties-content: |
+            GOROOT=/usr/local/go
+            GOPATH=$WORKSPACE/go
+            PATH=$PATH:$GOROOT/bin:$WORKSPACE/go/bin
+    - timeout:
+        timeout: 700
+        fail: true
+    - workspace-cleanup:
+        dirmatch: true
+        exclude:
+        - 'test-infra/.git/'
+        external-deletion-command: 'sudo rm -rf %s'
+
+- project:
+    jobs:
+    - 'ci-{soak-suffix}'
+    name: bootstrap-ci
```

Comment:
`jenkins_jobs.errors.JenkinsJobsException: Duplicate entry found in '/test-infra/jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-soak.yaml: 'bootstrap-ci' already defined`

this need to be bootstrap-ci-soak?